sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java

```diff
@@ -41,11 +44,32 @@ static void configureExporter(
             "opentelemetry-exporter-logging");
         configureLoggingMetrics(config, meterProvider);
         return;
+      case "none":
+        return;
       default:
+        MetricExporter spiExporter = configureSpiExporter(name, config);
```

Comment:
Are we ok that autoconfiguration only supports push based exporters using this method? Are there any other pull-based backends out there, aside from prometheus to worry about?

Reply:
Maybe the bigger question is: is this going to be good enough, given the requirements that multiple metric exporters running at the same time need to be supported by the SDK?

Reply:
So I looked at this.  MetricExporter *cannot* be a pull-based exporter.   Go take a gander at [prometheus exporter](https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusCollector.java#L15), it's not a `MetricExporter`. 

Effectively we have the Hierarchy:

- Pull-Based Exporters
  - Prometheus
  - IntervalMetricReader
     - MetricExporter
       - OTLP
       - InMemory
       - ... extensions...
       
I was worried about putting this behind IMR until I looked at the details.

Reply:
oh, yes, you are correct about that, technically. I. just to to make sure that putting this into the SPI is going to fulfill enough use-cases, and be future-proof enough to justify it, given the still ¯\_(ツ)_/¯ state of support for multiple exporters in the SDK.

Reply:
This is fully inline w/ autoconf for trace (which only allows ONE autoconfigured exporter), even if trace can do mulitple in the SDK.

I'd suggest this can get merged and if we do support multiple here it's an autoconf change.

Reply:
heh. https://github.com/open-telemetry/opentelemetry-java/issues/3443

Agree it's probably fine...I just want to make sure we've all thought it through. :)

Reply:
Yeah. I think you should push on that part of this PR.   I'd *love* to be able to solve multi-exporter support (for trace + metrics), but I think we're ok for now.  Should I open any bugs around multi-exporter support?

Reply:
> Yeah. I think you should push on that part of this PR. I'd _love_ to be able to solve multi-exporter support (for trace + metrics), but I think we're ok for now. Should I open any bugs around multi-exporter support?

do we know how this is going to work in the SDK yet? It's really easy for tracing/spans right now. Metrics is a different beast with the collection cycle(s), though.

Reply:
Right now there's been two major directions discussed:

 One direction is very close to what we have now where multi-exporter semantics are handled by something like IMR and pull-based receivers become (somewhat) stateful.  I think this is what Reiley showed us yesterday in the SiG.

The other option is the multi-collection idea that's in the prototype on my wip-metrics branch.   

I think the first option will be technically easier too maintain and cleaner.  Both approaches, I suspect, will work well with this exposed exporter interface (we just may have another SPI for pull-based exporters like prometheus).